docs/targets/http.md

```diff
@@ -0,0 +1,72 @@
+# Event Target for HTTP
+
+This event target receives [CloudEvents][ce] and turns them into HTTP request that consume external services.
+
+## Prerequisites
+
+The HTTP event target sends requests to arbitrary URLs and wraps responses in CloudEvents back to the caller. HTTP endpoints that are unauthenticated, use basic authentication or use custom header values for authentication, can be integrated using this target.
+
+Responses from external HTTP endpoints are converted into [CloudEvents][ce] and sent as a reply to the TriggerMesh Broker/Channel. It is important that the HTTP target filters received events and cares about response event type and event source to avoid loops where those responses might end up being processed by the HTTP Target.
+
+Requests from this HTTP target will verify TLS certificates from the remote server if present. If the CA certificate at the server is self-signed, the public certificate needs to be added to the configuration, or alternatively mark the skip verify option.
+
+If the remote endpoint requires basic authentication, the password needs to be created as a secret.
+
+## Create the HTTP Target Integration
+
+Create an instance of the HTTP Target at TriggerMesh as part of a Bridge.
+
+- `Name` is an internal identifier for the target.
+- `Response event type` must be set and will identify the type for responses from the remote endpoint.
+- `EventSource` will identify the origin for all responses. When not informed source will be automatically set to a generated name that includes the HTTP Target component name.
+- `Endpoint` full URL for the remote service, including path and query string if any.
+- `Method` to use when executing requests against the remote endpoint.
+- `CA Certificate`  CA certificate configured for TLS connection.
+- `Skip Verify` skips remote server TLS certificate verification.
+- `Username` when using basic authentication.
+- `Password` when using basic authentication needs to reference the aforementioned password secret.
+- `Headers` is a set of key/value pairs that will be set withing the HTTP request.
+
+Save the target, fill the rest of the bridge components and press `Submit Bridge`.
```

Comment:
...components`,` and press...

Reply:
https://github.com/triggermesh/docs/pull/78/commits/dbaa760053abaef48b4fc2e1d1bd82c437a572ab